packages/common/src/intl/locales/en/common.json

```diff
@@ -35,6 +35,7 @@
   "error.no-data": "No data available",
   "error.no-items": "No items",
   "error.no-items-filter-on": "No items to display. Try changing the filter criteria.",
+  "error.no-report-permission": "No permission to view reports",
```

Comment:
there's separate handling for the lack of print permission - that one will show the permission modal, which is good. This one is different, and shows that you can't even view a list of reports

Reply:
oh.. that's interesting! I was as one point able to query for reports.. but was getting a `Forbidden` response when printing! no idea how that is possible.. as you say, it's just the `Report` permission. If that ever does happen, it is handled nicely 😁  mostly though, users might not have permission to view reports, in which case they'll get the print button and a message saying that they don't have permission to view reports.